src/Sylius/Bundle/CoreBundle/Test/Services/SecurityService.php

```diff
@@ -45,14 +49,23 @@ public function __construct(
     /**
      * {@inheritdoc}
      */
-    public function logIn($email, $providerKey, Session $minkSession)
+    public function logIn($email, $providerKey = self::DEFAULT_PROVIDER_KEY, Session $minkSession)
```

Comment:
It is required. It is the name of the firewall [here](https://github.com/michalmarcinkowski/Sylius/blob/5009463405dee467e3503b210f3a30e274ec9414/app/config/security.yml#L51-L51).